core/datasource/external/ethverifier/verifier_snapshot.go

```diff
@@ -126,14 +156,22 @@ func (s *Verifier) LoadState(ctx context.Context, payload *types.Payload) ([]typ
 		s.restorePendingCallEvents(ctx, pl.EthContractCallEvent)
 		return nil, nil
 	case *types.PayloadEthOracleLastBlock:
-		s.restoreLastEthBlock(pl.EthOracleLastBlock)
+		s.restoreLastEthBlock(ctx, pl.EthOracleLastBlock)
+		return nil, nil
+	case *types.PayloadEthVerifierMisc:
+		s.restoreMisc(ctx, pl.Misc)
 		return nil, nil
 	default:
 		return nil, types.ErrUnknownSnapshotType
 	}
 }
 
 func (s *Verifier) OnStateLoaded(ctx context.Context) error {
+
+	if vgcontext.InProgressUpgradeFrom(ctx, "v0.74.8") {
```

Comment:
this will need to be 74.7, the version on mainnet is 74.7 and we will likely skip 74.8 in favour of a patch with this code